rfs_test/TEST_accountservice_schema.py

```diff
@@ -18,283 +18,419 @@
+
+        MaxPasswordLength = json_payload['MaxPasswordLength']
+        
+        authorization = 'on'
+        rq_headers = self.request_headers()
+        rq_body = {
+            'UserName': 'test_max',
+            'Password': ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(MaxPasswordLength + 1)),
+            'RoleId': 'Administrator'
+        }
+        json_payload, headers, status = self.http_GET(self.sut_toplevel_uris['AccountService']['url'], rq_headers, authorization)
+
+        # 201 is the only status when a request has created a new resource successfully
+
+        if status == 201:
+            assertion_status = log.FAIL
+            log.assertion_log('line', "Assertion Failed")
+            return assertion_status
 
-# NOT NECESSARY  
+    except:
+        assertion_status = log.WARN
+        log.assertion_log('line', "~ \'AccountsService\MaxPasswordLength' not found in the payload from GET %s" % (
+            '/redfish/v1/AccountService'))
+        return assertion_status
 
-###################################################################################################
-# Name: Assertion_ACCO111(self, log) :Account Service                                            
-# Assertion text: 
-# The Actions property shall contain the available actions for this resource.
-###################################################################################################
+    log.assertion_log(assertion_status, None)
+    log.assertion_log('line', "Assertion Passes")
+    return assertion_status
 
 
-## end Assertion_ACCO111
+# end Assertion_ACCO104
 
-# NOT NECESSARY  
 
 ###################################################################################################
-# Name: Assertion_ACCO112(self, log) :Account Service                                            
-# Assertion text: 
-# The Actions property shall contain the available actions for this resource.
+# Name: Assertion_ACCO105(self, log) :Account Service
+# Assertion text:
+# This property shall reference the threshold of failed login attempts at which point the user's
+# account is locked.  If set to 0, no lockout shall ever occur.
 ###################################################################################################
 
+def Assertion_ACCO105(self, log):
+
+    log.AssertionID = 'ACCO105'
+    assertion_status = log.PASS
+    log.assertion_log('BEGIN_ASSERTION', None)
+
+    relative_uris = self.relative_uris
+    authorization = 'on'
+    rq_headers = self.request_headers()
     
-## end Assertion_ACCO112
- 
-# NOT NECESSARY  
+    json_payload, headers, status = self.http_GET(self.sut_toplevel_uris['AccountService']['url'], rq_headers, authorization)
+
+    try:
+        AccountLockoutThreshold = json_payload['AccountLockoutThreshold']
+
+        if AccountLockoutThreshold == 0: 
+            assertion_status = log.WARN
```

Comment:
Should be "log.INFO" (or something similar)

Reply:
Fixed

Reply:
Fixed